dockerfiles/benchmarks/check_bench_result.py

```diff
@@ -3,69 +3,90 @@
 The script compares current benchmark result with the previous
 and creates an issue in GitHub if it exceeds the threshold
 Usage: check_bench_result.py <file with benchmark results> <github_org>
-Example: check_bench_result.py output.txt
+
+EXAMPLE:
+export CI_PROJECT_NAME=jsonrpsee
+export PROMETHEUS_URL="https://thanos.parity-mgmt.parity.io/"
+./check_bench_result.py output.txt
 """
 
 import os
 import sys
 from prometheus_api_client import PrometheusConnect
 from github import Github
 
+import urllib3
+
+urllib3.disable_warnings()
+
+
 try:
-    github_org = os.environ['GITHUB_ORG']
+    github_org = os.environ["GITHUB_ORG"]
 except KeyError:
     github_org = "paritytech"
 
 try:
     results_file = sys.argv[1]
-    github_token = os.environ['GITHUB_TOKEN']
-    github_repo = os.environ['CI_PROJECT_NAME']
-    github_repo_full = f"{github_org}/{github_repo}"
-    current_sha = os.environ['CI_COMMIT_SHA']
-    prometheus_url = os.environ['PROMETHEUS_URL']
+    prometheus_url = os.environ["PROMETHEUS_URL"]
+    github_repo = os.environ["CI_PROJECT_NAME"]
 except KeyError as error:
     print(f"{error} not found in environment variables")
     sys.exit(1)
 except IndexError:
     print("Please specify file with benchmark results and github_org")
-    print("Usage:",sys.argv[0],"<file with benchmark results>")
+    print("Usage:", sys.argv[0], "<file with benchmark results>")
     sys.exit(1)
 
 try:
-    threshold = os.environ['THRESHOLD']
+    threshold = os.environ["THRESHOLD"]
 except KeyError:
     threshold = 20
 
 prometheus_client = PrometheusConnect(url=prometheus_url, disable_ssl=True)
-github_client = Github(github_token)
 
-def benchmark_last_result(project,benchmark):
+# Allow script to run locally to check connection to Promethues without the
+# need of a github token
+try:
+    github_token = os.environ["GITHUB_TOKEN"]
+    github_repo_full = f"{github_org}/{github_repo}"
+    current_sha = os.environ["CI_COMMIT_SHA"]
+    github_client = Github(github_token)
+except Exception as e:
+    print(
+        f"WARNING: No github parameter {e} provided."
+        "GITHUB reporting wil not work."
+    )
+
+
+def benchmark_last_result(project, benchmark):
     """
     Get latest benchmark result from Victoria Metrics
     Returns "-1" if result not found
     """
-    query = f"last_over_time(parity_benchmark_common_result_ns{{project=\"{project}\",benchmark=\"{benchmark}\"}}[1y])"
+    query = f'last_over_time(parity_benchmark_common_result_ns{{project="{project}",benchmark="{benchmark}"}}[1y])'
     query_result = prometheus_client.custom_query(query=query)
     if len(query_result) > 0:
-        last_benchmark_result = int(query_result[0]['value'][1])
+        last_benchmark_result = int(query_result[0]["value"][1])
     else:
         last_benchmark_result = -1
     return last_benchmark_result
 
+
 def benchmark_last_sha(project):
     """
     Get short sha of latest benchmark result from Victoria Metrics
```

Comment:
```suggestion
    Get short sha of latest benchmark result from Thanos
```
Since we are getting results from Thanos